### PR TITLE
Fix localstack version + add required env var to api for tests + Add stagingWorkspaceType (PS-867)

### PR DESCRIPTION
**Pull request by JakubMatejka**

### CODE REVIEW

**Comment by JakubMatejka** [comment]
Aah sorry, it seems I have forgotten here the PR for fixing that broken localstack. Is it ok if I keep it in one PR?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -67,3 +68,6 @@ jobs:
           -e "KBC_STORAGE_TOKEN=${{ secrets.KBC_STORAGE_TOKEN }}" \
           -e "KBC_MANAGE_TOKEN=${{ secrets.KBC_MANAGE_TOKEN }}" \
           keboola/sandboxes-api-php-client composer test
+      - name: Dump docker logs on failure
+        if: failure()
+        uses: jwalton/gh-docker-logs@v1
```
Comment (pivnicek) [inline .github/workflows/build.yml:73]: oh, interesting!

**Review by pivnicek — APPROVED** [review]
pekne

File: .github/workflows/build.yml (hunk 1 above)
Comment (JakubMatejka) [reply .github/workflows/build.yml:73]: Yes, works great! It helped me to finally find out why the builds were still failing on CI.
image
I had a typo in the PROVIDER env var. 😄

**Pull request merged**